### PR TITLE
Adding iCatcher models

### PR DESCRIPTION
**Pull request by iCatcherAI**
I would like to add the following two iCatcher models:

- iCatcher Realistic
- iCatcher Cartoon

Metadata JSON, license (CreativeML Open RAIL++-M) and assets (example images) can be found in the newly created model folders. Not sure about the self-reported age rating though, as all three models are pretty much uncensored and therefore have the flexibility to create SFW and NSFW content, so should I choose 12+ or perhaps even 17+?

**Recommended Settings:**
- CLIP Skip: 1
- Text Guidance: 2
- Steps: 12
- Sampler: Euler A Trailing
- HiRes Fix: Upscale by 1.5 & 2nd Pass Strength @ 40%
- Optional: Detailer Script for Faces
- Optional: UniversalUpscaler V2 Sharp

**Recommended Positive Prompt:**

score_9, score_8_up, score_7_up
[YOUR PROMPT]

**Recommended Negative Prompt:**

Usually not needed

_**Disclaimer:** This is my first time using GitHub to contribute something, so I hope I've done everything the way it should be done._

### CODE REVIEW

File: models/icatcher-semireal/metadata.json [hunk 1]
```diff
@@ -0,0 +1,16 @@
+{
+  "download": {
+    "file": "https://civitai.com/api/download/models/556730",
```
Comment (liuliu) [inline models/icatcher-semireal/metadata.json:3]: Thanks for submission. A few things: first make sure this is accessible without login (I think so, just want to confirm).

File: models/icatcher-semireal/metadata.json [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+  },
+  "name": "iCatcher Semireal",
+  "file": "icatcher_semireal_f16.ckpt",
+  "prefix": "",
+  "modifier": "none",
+  "default_scale": 16,
+  "upcast_attention": false,
+  "version": "sdxl_base_v0.9",
+  "autoencoder": "sdxl_vae_v1.0_f16.ckpt",
+  "clip_encoder": "icatcher_semireal_sdxl_clip_vit_l14_f16.ckpt",
```
Comment (liuliu) [inline models/icatcher-semireal/metadata.json:14]: remove this and the one below. These will be auto-filled in when the model is converted. It is only useful if your fine-tune uses vanilla text encoders then you reference the vanilla one here.

File: models.txt [hunk 3]
```diff
@@ -1,3 +1,6 @@
+icatcher-realistic
```
Comment (liuliu) [inline models.txt:1]: move this to line 12 (before this change) where is at end of our sdxl community models.

File: models.txt [hunk 4]
```diff
@@ -1,3 +1,6 @@
...
+icatcher-semireal
+icatcher-cartoon
 lcm-sdxl-v1.0
```
Comment (liuliu) [inline models.txt:4]: note to myself, I will remove this one and the lcm-ssd-1b in favor of the lora.

**Review by liuliu — COMMENTED** [review]
Thanks! See the inline comment. Removing clip_encoder / text_encoder fields (and let the automated process to fill in) is important otherwise people won't be able to use the model properly.

**Comment by liuliu** [comment]
As for the rating, please add that to README and I will add that support at later time into metadata. Currently the app is rated at 17+ in AppStore so no worries about that for now.

File: models/icatcher-semireal/metadata.json (hunk 1 above)
Comment (iCatcherAI) [reply models/icatcher-semireal/metadata.json:3]: all my models should already be accessible without login.

**Review by iCatcherAI — COMMENTED** [review]
I hope I've made the correct changes based on your feedback (also reduced the models from three to just two). Keep up the great work (I also hope the implementation of SD3 doesn't cause too much headache). By the way, the integration of a donation button into the app is also long overdue; after all, I am certainly not the only one who would like to give something back.

**Pull request merged**